README.md

```diff
@@ -33,6 +33,26 @@ Several Lottie features are not yet supported, including:
 - Split rotations
 - Split positions
 
+## Usage
+
+Velato makes it simple to encode Lottie as a [`vello::Scene`](https://docs.rs/vello/*/vello/struct.Scene.html).
+
+```rust
+// Parse your lottie file
+let lottie = inclide_str!("../lottie.json");
+let composition = velato::Composition::from_str(lottie).expect("valid file");
+
+// Render to a scene
+let mut new_scene = vello::Scene::new();
+
+// Render to a scene!
+let renderer = velato::Renderer::new();
+let frame = 0.0; // Arbitrary number chosen. Ensure it's a valid frame!
+let transform = Affine::IDENTITY;
+let alpha = 1.0;
+renderer.render(composition, frame, transform, alpha, &mut new_scene);
+```
+
 ## Examples
```

Comment:
Is the `simple` example included in this README from before this PR?

Reply:
No, I added the simple example.

Reply:
I think it would be good to add it to the README

Reply:
Sure, we should do the same to the vello repo then.

Reply:
Ok, I see the simple example fix isn't planned for vello. I'm still thinking I don't want to include an example that doesn't compile for web, for now at least.